src/jobs/microk8s_install.yml

```diff
@@ -27,33 +27,30 @@ parameters:
   microk8s_values_content:
     description: "Content for microk8s-values.yaml, used to override chart default values"
     type: string
-    default: |
-      global:
-        imagePullSecrets:
-          - name: docker-credentials
-      listener:
-        persistence:
-          claim:
-            size: 8Gi
-        nfsserverprovisioner:
-          persistence:
-            size: 10Gi
-      minio:
-        persistence:
-          size: 10Gi
-      rabbitmq-ha:
-        replicaCount: 1
+    default: ""
+  microk8s_wait_retries:
+    description: "Number of retries to wait for microk8s to be ready"
+    type: integer
+    default: 20
+  kubectl_timeout:
+    description: "Time to wait for kubectl commands"
+    type: integer
+    default: 300
```

Comment:
We may or may not have to increase this value in the future. 
When we rotate pods, because we are manually doing so, we give kubeclt a --timeout of 5m or so.
This is just so we keep that in mind. No need to increase it now :)

Reply:
Thanks for the warning. I'll keep that in mind. In this case it might not be an issue because the "cluster" is always created from scratch. The whole step, including installation, generally runs in about 3 to 4 min.